src/editor/Store/WindowsStoreConnector.re

```diff
@@ -134,6 +134,7 @@ let start = getState => {
       }
     | AddSplit(direction, split) => {
```

Comment:
I tested this locally, and I think it is actually breaking the 'single-file' mode case.

When we load for the first time - we actually dispatch an `AddSplit` for the first editor group. So it seems like zen-mode is being disabled on launch, even when `editor.zenMode.singleFile` is true. 

I think we'll have to add a condition here to check if there is more than one window split - in other words, we don't want to disable zen mode if this is the first split being added.

Reply:
Fixed in https://github.com/onivim/oni2/pull/713/commits/8b634aa2ee1781b454fa549bd6810a95bb01cd70